src/config/configuration.ts

```
@@ -285,6 +288,11 @@ export default () => {
                 baseUrl: process.env.BIOMEDICAL_URL,
                 queryUrl: process.env.BIOMEDICAL_QUERY_URL,
             },
+            // 바이오식품공학과 추가 (2026.02.18)
```

ESLint

[error] Replace `············` with `······`

(prettier/prettier)